PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,30 @@
-Please make sure you've read the [pull request](http://docs.mattermost.com/developer/contribution-guide.html#preparing-a-pull-request) section of our [code contribution guidelines](http://docs.mattermost.com/developer/contribution-guide.html).
+<!-- Thanks contribution a pull request! Here are some tips for you:
```

Comment:
Propose `Thank you for contributing a pull request! Here are a few tips to help you: